redbot/core/tree.py

```diff
@@ -331,6 +332,9 @@ async def on_error(
             await self._send_from_interaction(
                 interaction, _("You are not permitted to use this command.")
             )
+        elif isinstance(error, UserFeedbackCheckFailure):
+            if error.message:
+                await self._send_from_interaction(interaction, error.message)
```

Comment:
This must be before the `CheckFailure` branch, or that branch will trigger first due to `UserFeedbackCheckFailure` being a subclass.